src/commands/whois.js

```diff
@@ -16,41 +16,53 @@ module.exports = {
     ],
     execute: async ({ interaction, response }) => {
         // Get the raw values from Discord
-        const query = ((interaction.data.options.find(opt => opt.name === 'query') || {}).value || '').trim()
-            .replace(/^as([0-9]+)$/i, '$1') // Remove 'AS' from start of an ASN
-            .replace(/^[a-z][a-z0-9+.-]+:\/\/(.+)$/i, '$1') // Remove scheme from a URI
-            .replace(/^([0-9]{1,3}(?:\.[0-9]{1,3}){3}):[0-9]+$/, '$1'); // Remove port from an IPv4 address
+        const query = ((interaction.data.options.find(opt => opt.name === 'query') || {}).value || '')
+            .trim()
+            // Remove 'AS' from start of an ASN
+            .replace(/^as([0-9]+)$/i, '$1')
+            // Remove scheme from a URI
+            .replace(/^[a-z][a-z0-9+.-]+:\/\/(.+)$/i, '$1')
+            // Remove port from an IPv4 address
+            .replace(/^([0-9]{1,3}(?:\.[0-9]{1,3}){3}):[0-9]+$/, '$1');
 
         // TODO: Try to validate as domain/IPv4/IPv6/ASN before running lookup
 
         // Do the rdap/whois lookup
         const data = await performLookup(query);
 
         // If no result, send back simple message
-        if (!data)
-            return response({
-                type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
-                data: {
-                    embeds: [createEmbed('WHOIS',
-                        'The query does not appear to be a valid domain name, IP address or ASN, or no results could be found')],
-                },
-            });
+        if (!data) return response({
+            type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+            data: {
+                embeds: [
+                    createEmbed(
+                        'WHOIS',
+                        'The query does not appear to be a valid domain name, \
+                        IP address or ASN, or no results could be found'
```

Comment:
This introduces a load of whitespace into the string?

Reply:
I don't think it will, i'm pretty sure the indents are only an issue if you're using *template literals*. Looking online, this is the only clean way to create multi-line strings without string concat.

Reply:
This absolutely does introduce a load of whitespace into the string...

```
const originalStr = 'The query does not appear to be a valid domain name, IP address or ASN, or no results could be found';

const newStr = 'The query does not appear to be a valid domain name, \
                        IP address or ASN, or no results could be found';

console.log(originalStr.length);
console.log(newStr.length);
```

```
100
124
```

If the string is too long to fit in a single line with the current eslint rules, then update the eslint config to allow strings to be longer -- there is a `"ignoreStrings": true` option for `max-len`.